packages/react/test-app/cypress/integration/issues/gh-25590.spec.ts

```diff
@@ -0,0 +1,41 @@
+describe('IonModal', () => {
```

Comment:
Could we name this file something more descriptive, maybe `conditional-render.spec.ts` or something? I would rather us not have to dig through the code and/or click through to the GH issue to figure out the context.